pkg/cmd/workspace/code.go

```diff
@@ -124,7 +144,24 @@ var CodeCmd = &cobra.Command{
 		}
 
 		if !workspace_util.IsProjectRunning(workspace, projectName) {
-			views.RenderInfoMessage(fmt.Sprintf("Project '%s' from workspace '%s' is not in running state", projectName, workspace.Name))
+			if !autoStartFlag {
```

Comment:
The block of code from !autoStartFlag till openIDE gets repeated twice.
I'd prefer to extract a workspace starting function similarly to this review: https://github.com/daytonaio/daytona/pull/1004#discussion_r1735912448

This will avoid code repeating and running unnecessary checks due to "StartCmd.Run" calls.
For OpenTerminalSsh we can just send "ssh" as ideId and nil for providerMetadata, "RenderInfoMessage" should get a boolean for displaying the message or not. This way almost all the code can be reused as well.